openhands/server/routes/git.py

```diff
@@ -16,31 +16,25 @@
     User,
 )
 from openhands.server.auth import get_access_token, get_provider_tokens
+from openhands.server.types import AppMode
 
 app = APIRouter(prefix='/api/user')
 
 
-from pydantic import BaseModel
-
-
 @app.get('/repositories', response_model=list[Repository])
 async def get_user_repositories(
     sort: str = 'pushed',
-    installation_id: int | None = None,
+    app_mode: AppMode = AppMode.OSS,
```

Comment:
it's odd to me that we pass this around as a function argument, since it's globally set. Is there a more elegant way to check this as global state?

Reply:
actually using the var directly from the BE causes pretty terrible circular import issues 😅